src/journal/journald.c

```diff
@@ -52,7 +52,7 @@ int main(int argc, char *argv[]) {
                 goto finish;
 
         server_vacuum(&server, false);
-        server_flush_to_var(&server);
+        server_flush_to_var(&server, true);
```

Comment:
Well, I tested this branch. I'm seeing:
```
-rw-r-----+ 1 root root 4845568 Dec 16 06:40 system.journal
```
So, actually, https://github.com/systemd/systemd/pull/2057/commits/6b95e47067e52be60b37307d57f1ed2b5419b9d5 fixes #1397. I think the commit message shouldn't contain "Fixes #1397"